packages/donate-button-beta/src/components/widget/Frequency/index.tsx

```diff
@@ -0,0 +1,104 @@
+import cxs from 'cxs';
+import {useState} from 'preact/hooks';
+import {Fragment} from 'preact/jsx-runtime';
+import {COLORS} from 'src/components/widget/theme/colors.enum';
+
+const frequencyContainerCss = cxs({
+	display: 'flex'
+});
+
+const labelCss = cxs({
+	color: COLORS.Primary,
+	fontSize: '1rem',
+	lineHeight: 1.5,
+	fontWeight: 400,
+	letterSpacing: '-0.01em',
+	padding: '8px 0px',
+	flex: 1,
+	textAlign: 'center',
+	border: '1px solid',
+	borderColor: COLORS.LightGray,
+	transition: 'all .2s'
+});
+
+const labelSelectedCss = cxs({
+	borderColor: COLORS.Primary
+});
+
+const separatorBorderSelectedCss = cxs({
+	borderRightColor: COLORS.Primary
+});
+
+const labelLeftCss = cxs({
+	borderRadius: '0.5rem 0 0 0.5rem'
+});
+
+const labelRightCss = cxs({
+	borderRadius: '0 0.5rem 0.5rem 0',
+	borderLeft: 'none'
+});
+
+const inputCss = cxs({
+	display: 'none'
+});
+
+const titleCss = cxs({
+	margin: 0,
+	fontSize: '1rem',
+	lineHeight: 1.5,
```

Comment:
We also should standardize font sizes, we can again use the same values in the Every.org design system